lib/node_modules/@stdlib/utils/any-in-by/docs/repl.txt

```diff
@@ -0,0 +1,41 @@
+{{alias}}( collection, predicate[, thisArg ] )
```

Comment:
REPL documentation here needs to be updates as it currently is written for `collection` / `array` inputs, but the function is for `object`s.